source/javascripts/controllers/_MainController.js.erb

```diff
@@ -488,9 +489,21 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 									return appService.saveSecrets().then(resolve, reject);
 								case "stack":
 									viewModel.dataToSave = appService.appConfig;
+									var machineTypeUpdated = appService.defaultMachineTypeHasUnsavedChanges();
+
 									return appService
 										.saveStackAndDockerImage(deferredSavePromise, viewModel.currentMenu.id)
 										.then(function() {
+						
+											if (machineTypeUpdated) {
+												const event = 'Machine Type Changed - Stacks & Machines Tab';
+												window.analytics && window.analytics.track(event, {
+												  machine_type_id: viewModel.dataToSave.meta['bitrise.io'].machine_type_id,
+												  user_slug: viewModel.currentUser.slug,
+												  organization_slug: appService.appDetails.ownerData.type === 'Organization' ? appService.appDetails.ownerData.slug: undefined
```

Comment:
A space is missing before the `:` in the ternary :)

Reply:
It's completely blank here for me, but if you go to the Files Changed tab it shows the correct line :)

Reply:
This has been updated